statistics/handle/gc_test.go

```diff
@@ -15,16 +15,48 @@
 package handle_test
 
 import (
+	"fmt"
+	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/domain"
 	"github.com/pingcap/tidb/sessionctx/variable"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/stretchr/testify/require"
 )
 
-func (s *testStatsSuite) TestGCStats(c *C) {
-	defer cleanEnv(c, s.store, s.do)
-	testKit := testkit.NewTestKit(c, s.store)
+func createTestKitAndDom(t *testing.T) (*testkit.TestKit, *domain.Domain, func()) {
```

Comment:
If you create new store for each test case, I think you don't have to implement `statsClean`.

Reply:
ok, get it.
the current problem is that i can not find the reason why it fails after migrate, need more time.

Reply:
I'm trying to run the test before migrated, it gives

* failed if run only the test
* success if run the whole package

I think there must be some share state in the whole package. Maybe @winoros has more ideas on it.

Reply:
thank you for help checking, @tisonkun . 
i checked carefully myself and believe i only did simple one - one replacement migrate, not sure why it fails and have to dig more to find the reason.
although this is a good chance to understand the logic of statistics(actually i read a lot), guide to how to locate the reason is still appreciate。@winoros